packages/core/chrome/core-chrome-browser-internal/src/chrome_service.tsx

```diff
@@ -225,6 +225,11 @@ export class ChromeService {
       projectNavigation.setProjectHome(homeHref);
     };
 
+    const setProjectsUrl = (projectsUrl: string) => {
```

Comment:
I wonder if at some point replacing all the setters with a single one would make sense... Maybe not as that function would be a big one instead of many small functions... 🤔 

```ts
updateProjectConfig({ ... partial configs here });
```